controls/radpdfviewer/features/bookmarks.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Bookmarks (Outlines)
+page_title: Bookmarks (Outlines)
+description: Bookmarks (Outlines)
+slug: radpdfviewer-features-bookmarks
+tags: bookmarks, outlines
+published: True
+position: 0
+---
+
+# Bookmarks (Outlines)
+
+RadPdfViewer enables you to open documents and show the bookmarks they contain in a bookmarks panel. Bookmarks or Outlines are a tree-structured hierarchy that the reader presents as a visual table of contents separated from the actual content as a side panel. The Bookmark items allow users, by interacting with them, to navigate through parts of the document and/or invoke different actions.  
+
+>The Bookmarks (Outlines) functionality is available as of R3 2018. 
+
+
+#### Figure 1: Bookmarks in RadPdfViewer
+
+![](images/RadPdfViewer_Bookmarks_01.png)
+
+
+## Visualizing Bookmarks
+
+In **RadPdfViewer**, you can show bookmarks using **RadPdfViewerNavigationPane**. This is a control representing an area that can display different navigation panels, including the Bookmarks of the PDF document which serve as a table of contents. RadPdfViewerNavigationPane exposes the **PaneWidth** property, which you can use to specify the width of the panel.
+
+#### [XAML] Example 1: Define RadPdfViewerNavigationPane
+
+{{region radpdfviewer-features-bookmarks_0}}
+
+	<Grid>
+	    <Grid.ColumnDefinitions>
+	        <ColumnDefinition Width="Auto" />
+	        <ColumnDefinition />
+	    </Grid.ColumnDefinitions>
+	    <telerik:RadPdfViewerNavigationPane RadPdfViewer="{Binding ElementName=pdfViewer, Mode=OneWay}" PaneWidth="300" />
```

Comment:
Let's change the binding type to OneTime? It's more safe and is the code included in the code generation and in the demos.